SalesforceCommerceCloud/pwa-kit
# Simplified homepage for Retail React App

GUS: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000GZZ3OYAX/view

# Description

This PR implements the home page design detailed in the [Figma file](https://www.figma.com/file/gOUKasMrK16zS5qrLCTSJl/PWA-Design-Kit?node-id=11554%3A200140).

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Updated home page.
- Updated `<Hero />` and `<Section />` components.
- Removed old images files. 

# How to Test-Drive This PR

- Check that the implementation matches the [Figma design](https://www.figma.com/file/gOUKasMrK16zS5qrLCTSJl/PWA-Design-Kit?node-id=11554%3A200140).
- Check the links point to the proper URLs detailed in [GUS](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000GZZ3OYAX/view).

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
